aspnetcore/host-and-deploy/iis/development-time-iis-support.md

```diff
@@ -28,11 +28,11 @@ Enable IIS. Navigate to **Control Panel** > **Programs** > **Programs and Featur
 
 ![Windows Features showing Internet Information Services checkbox checked as a black square (not a checkmark) indicating that some of the IIS features are enabled](development-time-iis-support/_static/enable_iis.png)
 
-If the IIS installation requires a reboot, reboot the system.
+If the IIS installation requires a restart, restart the system.
 
 ## Enable development-time IIS support
 
-Once IIS is installed, launch the Visual Studio installer to modify the existing Visual Studio installation. In the installer, select the **Development time IIS support** component. The component is listed as an optional component in the **Summary** panel for the **ASP.NET and web development** workload. This installs the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module), which is a native IIS module required to run ASP.NET Core applications.
+Launch the Visual Studio installer. Select the **Development time IIS support** component. The component is listed as an optional component in the **Summary** panel for the **ASP.NET and web development** workload. This installs the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module), which is a native IIS module required to run ASP.NET Core apps.
```

Comment:
~~The component is listed as an optional component~~
The component is listed as optional